http/obmc_shell.hpp

```diff
@@ -0,0 +1,242 @@
+#pragma once
+
+#include <pty.h>
+#include <pwd.h>
+#include <termios.h>
+
+#include <app.hpp>
+#include <boost/asio/posix/stream_descriptor.hpp>
+#include <boost/process/io.hpp>
+#include <websocket.hpp>
+
+#include <map>
+
+namespace crow
+{
+
+namespace obmc_shell
+{
+
+class Handler : public std::enable_shared_from_this<Handler>
+{
+  public:
```

Comment:
Do you need a max sessions? 4 ? 10 ?

Reply:
right now there is no limit defined. 
do we need a limit?